### PR TITLE
CASMINST-6228 Add diagrams for IUf installs and upgrades

### PR DESCRIPTION
**Pull request by deantroe**
Add diagrams for IUf installs and upgrades that correspond to the instructions already provided.

# Description

CASMINST-6228

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by atifsyedali — APPROVED** [review]

**Review by don-bahls-hpe — APPROVED** [review]

**Review by JohnNavitskyHPE — APPROVED** [review]

File: operations/iuf/workflows/install_or_upgrade_additional_products_with_iuf.md [hunk 1]
```diff
@@ -15,8 +15,10 @@ recipe release is deployed and all management NCNs and managed compute and appli
 rebooted to new images and CFS configurations. Manual operations are documented for procedures that
 are not currently managed by IUF.
 
-The initial install workflow comprises the following procedures which must be executed in the order
-shown.
+The initial install workflow comprises the following procedures. The diagram visualizes the workflow and
```
Comment (sfkramer) [inline operations/iuf/workflows/install_or_upgrade_additional_products_with_iuf.md:18]: Replace "visualizes" with "displays" or "shows".  

"Visualizes" indicates the reader would have to read/examine our content and then create a mental image for themselves.

File: operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md [hunk 2]
```diff
@@ -21,7 +21,10 @@ recipe release is deployed and all management NCNs and managed compute and appli
 rebooted to new images and CFS configurations. Manual operations are documented for procedures that
 are not currently managed by IUF.
 
-The upgrade workflow comprises the following procedures which must be executed in the order shown.
+The upgrade workflow comprises the following procedures. The diagram visualizes the workflow and
```
Comment (sfkramer) [inline operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md:24]: Replace "visualizes" with "displays" or "shows".  

"Visualizes" indicates the reader would have to read/examine our content and then create a mental image for themselves.

**Review by sfkramer — CHANGES_REQUESTED** [review]
Just two one-word updates needed.

File: operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md (hunk 2 above)
Comment (deantroe) [reply operations/iuf/workflows/upgrade_csm_and_additional_products_with_iuf.md:24]: Fixed.

**Pull request merged**